Mu2eG4/src/constructDiskCalorimeter.cc

```diff
@@ -225,14 +225,9 @@ namespace mu2e {
 
        const bool isPipeVisible          = geomOptions->isVisible("calorimeterPipe"); 
        const bool isPipeSolid            = geomOptions->isSolid("calorimeterPipe"); 
-       const bool forceEdge              = geomOptions->forceAuxEdgeVisible("calorimeterPipe"); 
-       const bool doSurfaceCheck         = geomOptions->doSurfaceCheck("calorimeterPipe");
+       const bool forceEdge              = config.getBool("g4.forceEdge",false);
+       const bool doSurfaceCheck         = config.getBool("g4.doSurfaceCheck",false) || config.getBool("calorimeter.doSurfaceCheck",false);
        const int  verbosityLevel         = config.getInt("calorimeter.verbosityLevel",1);
-
-       if ( verbosityLevel > 0) {
-         G4cout << __func__ << " Called. doSurfaceCheck: " << doSurfaceCheck << G4endl;
-       }
-
```

Comment:
What was the reason to remove the guarded printout?

Reply:
will revert